<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QWidget" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="generalPage">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QCheckBox" name="launchOnStart">
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="text">
                  <string>Launch on startup</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="autoUpdate">
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="text">
                  <string>Auto-connection with unprotected wi-fi</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="notification">
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="text">
                  <string>Automatically update</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>100</height>
                </size>
               </property>
               <property name="title">
                <string/>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="1" column="1">
                 <widget class="QLabel" name="daysTrial">
                  <property name="text">
                   <string>4 days</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="freeTrial">
                  <property name="text">
                   <string>Free trial:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0" colspan="2">
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="3" column="0">
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="2" column="0" colspan="2">
                 <widget class="QLabel" name="upgradeLabel">
                  <property name="text">
                   <string>UPGRADE</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="connectionPage">
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QGridLayout" name="gridLayout_6">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Auto - Connect to</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>most recent location</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" rowspan="2">
              <widget class="QCheckBox" name="selectServerLocation">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>a server near you</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2" rowspan="5">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Connection mode:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="connectionMode"/>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="unprotectedWifi">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="unprotectedLabel">
               <property name="text">
                <string>With unprotected wi-fi</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="headerFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="styleHint" stdset="0">
      <string>header</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QToolButton" name="general">
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="styleHint" stdset="0">
         <string>tab</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="connection">
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="styleHint" stdset="0">
         <string>tab</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="blackList">
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="styleHint" stdset="0">
         <string>tab</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="wifi">
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="styleHint" stdset="0">
         <string>tab</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="account">
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="styleHint" stdset="0">
         <string>tab</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="close">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="horizontalWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="versionLabel">
        <property name="text">
         <string>Version: %1</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="save">
        <property name="text">
         <string>SAVE</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
